### DISCUSSION TITLE
Shutdown is triggered when the ONLINE state changes to the ONBATT state

### DISCUSSION THREAD

**Discussion by biubiushaka** [nutdotnet/WinNUT-Client#146] [Ideas]
I would like to add an option to trigger a shutdown action after changing from ONLINE to ONBATT, and there is also a "Delayed Shutdown" option.

**Comment by gbakeman** [comment]
Hi @biubiushaka ,

Is there any reason that a configuration like this wouldn't work for you?

![image](https://github.com/nutdotnet/WinNUT-Client/assets/842942/5fcfb84d-259b-4456-863f-9f2620cf6f52)
